Client/Helpers/FxALoginHelper.swift

```diff
@@ -90,8 +96,6 @@ class FxALoginHelper {
         // By the time we reach here, we haven't registered for APNS
         // Either we've never asked the user, or the user declined, then re-enabled
         // the notification in the Settings app.
-
-        self.account = account
         requestUserNotifications(application)
```

Comment:
This will trigger:

```func application(_ application: UIApplication, didRegister notificationSettings: UIUserNotificationSettings) {
    FxALoginHelper.sharedInstance.application(application, didRegisterUserNotificationSettings: notificationSettings)
}```
 